nntrainer/tensor/tensor_pool.h

```diff
@@ -328,7 +328,7 @@ class TensorPool {
   std::vector<RequestSpec> pool; /**< list of requested tensors */
   std::unordered_map<std::string, unsigned int>
     name_map;          /**< indexing of requested tensors */
-  MemoryPool mem_pool; /**< memory pool for the tensors */
+  CachePool mem_pool;  /**< memory pool for the tensors */
```

Comment:
I wonder do we have to change this? ```CachePool``` inherit ```MemoryPool``` and does it work if we turn the swap option off?

Reply:
It modified in later commit.
I removed this chages from the commit.